src/viewer.js

```diff
@@ -2735,42 +2735,54 @@ function onCanvasPinch( event ) {
     return false;
 }
 
+var lastScroll = new Date().getTime();
```

Comment:
I'm wanting to move away from having variables hidden inside our closure. Please make this an instance variable of the Viewer, and prefix it with an underscore to mark that it's "private". Also, give it a name that marks it as time based. So, something like: `this._lastScrollTime`.

Also, our preferred way to get the current milliseconds is `$.now()` instead of `new Date().getTime()`; please use that.